src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -184,13 +190,44 @@ const formConfig = {
     certifierInformation: {
       title: 'Signer information',
       pages: {
+        page1: {
+          // initialData: mockData.data,
+          path: 'signer-type',
+          title: 'Which of these best describes you?',
+          uiSchema: {
+            ...titleUI('Your information'),
+            certifierRole: radioUI({
+              title: 'Which of these best describes you?',
+              required: () => true,
+              labels: {
+                applicant: 'I’m applying for benefits for myself',
+                sponsor:
+                  'I’m a Veteran applying for benefits for my spouse or dependents',
+                other:
+                  'I’m a representative applying for benefits on behalf of someone else',
+              },
+              // Changing this data on review messes up the ad hoc prefill
+              // mapping of certifier -> applicant|sponsor:
+              hideOnReview: true,
+            }),
+          },
+          schema: {
+            type: 'object',
+            required: ['certifierRole'],
+            properties: {
+              titleSchema,
+              certifierRole: radioSchema(['applicant', 'sponsor', 'other']),
+            },
+          },
+        },
```

Comment:
This page collects the signer's `certifierRole`, which is the key to determining which data to duplicate + which extra screens to show.